app/paths.js

```diff
@@ -163,11 +163,14 @@ module.exports = {
     update: '/service/:externalServiceId/edit-name'
   },
   merchantDetails: {
-    index: '/organisation-details/:externalServiceId',
-    edit: '/organisation-details/edit/:externalServiceId'
+    index: '/service/:externalServiceId/organisation-details',
```

Comment:
Do we want these paths to be organised into `service` similar to the `account` paths?
```js 
{
  service: { },
  account: {}
}
```

How paths are organised wont change the functionality so totally fine either way

Reply:
They will be in the PR to add the router, just bodged in this change for now so didn't have to change things twice.